### PR TITLE
Clojure helloworld serving sample

### PR DESCRIPTION
**Pull request by josephburnett**
## Proposed Changes

* Adds a serving sample `helloworld-clojure`.

### CODE REVIEW

**Comment by josephburnett** [comment]
/assign evankanderson 

Hey Evan, can you review a Lisp sample?

**Comment by evankanderson** [comment]
/lgtm
/approve

I don't speak Clojure, but this looks about right.

File: serving/samples/helloworld-clojure/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+COPY . /usr/src/app
+
+# Build the app as an uberjar.
+RUN mv "$(lein uberjar | sed -n 's/^Created \(.*standalone\.jar\)/\1/p')" app-standalone.jar
+
+# Run the app by default when the container starts.
+CMD ["java", "-jar", "app-standalone.jar"]
+
+# Document that the service listens on port 8080.
+EXPOSE 8080
```
Comment (steren) [inline serving/samples/helloworld-clojure/Dockerfile]: You should not hardcode 8080, instead listen on $PORT, an env var injected by Knative.

File: serving/samples/helloworld-clojure/README.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+      {:status 200
+       :headers {"Content-Type" "text/html"}
+       :body (str "Hello World: "
+                  (if-let [target (System/getenv "TARGET")]
+                    target
+                    "NOT SPECIFIED")
+                  "!\n")})
+
+    (defn -main [& args]
+      (run-jetty handler {:port 8080}))
```
Comment (steren) [inline serving/samples/helloworld-clojure/README.md]: You should listen on $PORT, not 8080

**Pull request merged**